config/core/test-configs-chromeos.yaml

```diff
@@ -223,6 +223,30 @@ device_types:
         image: 'https://storage.chromeos.kernelci.org/images/rootfs/chromeos/chromiumos-hatch/20220811.0/amd64/bzImage'
         modules: 'https://storage.chromeos.kernelci.org/images/rootfs/chromeos/chromiumos-hatch/20220811.0/amd64/modules.tar.xz'
 
+  asus-C523NA-A20057-coral_chromeos:
+    base_name: asus-C523NA-A20057-coral
+    mach: x86
+    arch: x86_64
+    boot_method: depthcharge
+    filters: *pineview-filter
+    params:
+      <<: *chromebook-generic-params
+      cros_flash_nfs:
```

Comment:
After using chromebook-generic-params you can remove crosh_flash_nfs section from device, as it is "derived" from chromebook-generic-params.
For example look to hp-11A-G6-EE-grunt_chromeos and hp-x360-12b-ca0010nr-n4020-octopus_chromeos

Reply:
Updated.